client/lib/common/src/data/services.js

```diff
@@ -35,6 +36,10 @@ const SERVICES = {
         url: PIERONE_BASE_URL,
         root: '/v1/search',
         searchQuery: 'q'
+    },
```

Comment:
Ya, we don't need this here... Was a lame attempt to abstract link construction to the other services.